tests/test_lock.c

```diff
@@ -35,6 +35,7 @@
 
 #include "tests/config.h"
 #include "sysrepo.h"
+#include "utils.h"
```

Comment:
You forgot to remove this.

Reply:
sorry,it's my careless,I will modify it immediately.